modules/account-lib/src/coin/hbar/transactionBuilder.ts

```diff
@@ -65,7 +68,7 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     } else {
       buffer = rawTransaction;
     }
-    tx.bodyBytes(buffer);
+    tx.fromBytes(buffer);
```

Comment:
There should be a constructor so you can do `const tx = new Transaction(this._coinConfig, buffer)`